web/src/lib/static/loadRenewalCalendarEvents.ts

```diff
@@ -0,0 +1,37 @@
+import { XrayRenewalCalendarEventType } from "@/lib/types/types";
+import { convertXrayRenewalCalendarEventMd } from "@/lib/utils/markdownReader";
+import fs from "fs";
+import path from "path";
+import { loadUrlSlugByFilename } from "./helpers";
+
+type PathParams<P> = { params: P; locale?: string };
+export type XrayRenewalCalendarEventUrlSlugParam = {
+  xrayRenewalCalendarEventUrlSlug: string;
+};
+
+const renewalCalendarEventsDir = path.join(
+  process.cwd(),
+  "..",
+  "content",
+  "src",
+  "renewal-calendar-events",
+);
+
+export const loadXrayRenewalCalendarEvent = (): XrayRenewalCalendarEventType => {
```

Comment:
dust - Is this Xray specific? Or is it a general renewal? - See Filename vs func name